util/tls/tls_socket.h

```diff
@@ -127,26 +128,23 @@ class TlsSocket final : public FiberSocketBase {
 
     iovec scratch_iovec;
 
+    bool should_read = false;
+
     // Asynchronous helpers
     void MaybeSendOutputAsyncWithRead();
+    void MaybeSendOutputAsync();
 
     void HandleOpAsync();
+    void HandleUpstreamAsyncWrite(io::Result<size_t> write_result, Engine::Buffer buffer);
```

Comment:
naming. HandleUpstreamAsyncWrite is always used as a completion callback for AsyncWriteSome, 
so lets call it as CompleteAsyncWrite like CompleteAsyncReq